waypoints:

  n_layers: 1

  layer0:
    point0: [-0.274, -0.715, 1.625, 0.0, 0.0, -90.0]